x/aswap/msg.go

```diff
@@ -0,0 +1,111 @@
+package aswap
+
+import (
+	"github.com/iov-one/weave"
+	coin "github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/migration"
+)
+
+func init() {
+	// Migration needs to be registered for every message introduced in the codec.
+	// This is the convention to message versioning.
+	migration.MustRegister(1, &CreateSwapMsg{}, migration.NoModification)
+	migration.MustRegister(1, &ReleaseSwapMsg{}, migration.NoModification)
+	migration.MustRegister(1, &ReturnSwapMsg{}, migration.NoModification)
+}
+
+const (
+	pathCreateSwap      = "swap/create"
```

Comment:
q: why `swap`  when the package is `aswap`?

Reply:
Because we already have escrow and I couldn't find a better name for it to be honest.